addons/project/migrations/12.0.1.1/post-migration.py

```diff
@@ -19,8 +19,10 @@ def fill_project_project_inherits_values(cr):
 @openupgrade.migrate()
 def migrate(env, version):
     fill_project_project_inherits_values(env.cr)
-    openupgrade.load_data(
-        env.cr, 'project', 'migrations/12.0.1.1/noupdate_changes.xml')
+    task_14 = env.ref('project.project_task_data_14', False)
+    if task_14:
+        openupgrade.load_data(
```

Comment:
This is not the way to do this, but use this parameter instead:

https://github.com/OCA/openupgradelib/blob/b4214ff10ba1ca1fa3f0fd79f049abbba1874d68/openupgradelib/openupgrade.py#L272

that we explicitly added in openupgradelib for these cases